ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -154,9 +205,53 @@ def test_9_pwpolicy_mod(self):
         """
         Test the `xmlrpc.pwpolicy_mod` method.
         """
-        entry = api.Command['pwpolicy_mod'](self.group, krbminpwdlife=50)['result']
+        entry = api.Command['pwpolicy_mod'](self.group,
+                                            krbminpwdlife=50)['result']
         assert_attr_equal(entry, 'krbminpwdlife', '50')
 
+    def test_maxlife_pwpolicy(self):
+        """Check maxlife error message where minlife > maxlife specified
+
+        When minlife > maxlife specified on commandline, it says:
+        "ipa: ERROR: invalid 'maxlife': Maximum password life must be
+        greater than minimum."
+
+        But when minlife == maxlife specfied, It works.
+        This test check that error message says what exactly it does.
+
+        related: https://pagure.io/freeipa/issue/9038
+        """
+        # test pwpolicy_mod
+        # create a test group
+        test_group101 = 'testgroup101'
+        test_group102 = 'testgroup102'
+
+        self.failsafe_add(
+            api.Object.group,
+            test_group101,
+            description=u'pwpolicy test group',
+        )
+        self.failsafe_add(
+            api.Object.group,
+            test_group102,
+            description=u'pwpolicy test group',
+        )
+        # when minlife(specified in hours) == maxlife(specified in days)
+        pwpolicy_cmd('pwpolicy_mod', 24)
+        pwpolicy_cmd('pwpolicy_add', 24, 5, test_group101, pwpolicyadd=True)
+
+        # when minlife(specified in hours) < maxlife(specified in days)
+        pwpolicy_cmd('pwpolicy_mod', 20)
+        pwpolicy_cmd('pwpolicy_add', 20, 6, test_group102, pwpolicyadd=True)
+
+        # when minlife(specified in hours) > maxlife(specified in days)
+        pwpolicy_cmd('pwpolicy_mod', 25)
+        pwpolicy_cmd('pwpolicy_add', 25, 7, test_group101, pwpolicyadd=True)
+
+        # when minlife is -1
+        pwpolicy_cmd('pwpolicy_mod', -1)
+        pwpolicy_cmd('pwpolicy_add', -1, test_group101, pwpolicyadd=True)
```

Comment:
The cospriority argument is missing: the method expects `cmd, minlife, cospriority=None, pwpolicy_group=None, pwpolicyadd=False`.